WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/InitializationManager.java

```diff
@@ -111,5 +115,10 @@ public EventManager getEventManager() {
     public IntervalTransferManagerBase getTransferManager() {
         return transferManager;
     }
+
+    /** Adds a new listener for debug events. */
+    public void addDebuggerListener(Project project) {
```

Comment:
I would, but this will require some extra changes as you'll need a handle to a Project instead of only the name of the project